src/test/java/uk/nhs/hee/tis/trainee/forms/service/LtftServiceTest.java

```diff
@@ -148,4 +162,145 @@ void shouldCountFilteredLtftWhenFiltersNotEmpty() {
     assertThat("Unexpected count.", count, is(40L));
     verify(ltftRepository, never()).count();
   }
+
+  @Test
+  void shouldReturnEmptyIfLtftFormNotFound() {
+    when(ltftRepository.findByTraineeTisIdAndId(any(), any())).thenReturn(Optional.empty());
+
+    Optional<LtftFormDto> formDtoOptional = service.getLtftForm(ID);
+
+    assertThat("Unexpected form returned.", formDtoOptional.isEmpty(), is(true));
+    verify(ltftRepository).findByTraineeTisIdAndId(any(), eq(ID));
+    verifyNoMoreInteractions(ltftRepository);
+  }
+
+  @Test
+  void shouldReturnEmptyIfLtftFormForTraineeNotFound() {
+    when(ltftRepository.findByTraineeTisIdAndId(TRAINEE_ID, ID)).thenReturn(Optional.empty());
+
+    Optional<LtftFormDto> formDtoOptional = service.getLtftForm(ID);
+
+    assertThat("Unexpected form returned.", formDtoOptional.isEmpty(), is(true));
```

Comment:
I'd prefer `assertThat("Unexpected form returned.", formDtoOptional, is(Optional.empty()));` so we get a better failure message